drivers/led/lp5562.c

```diff
@@ -921,18 +925,72 @@ static int lp5562_led_update_current(const struct device *dev)
 	return ret;
 }
 
+static int lp5562_enable(const struct device *dev)
+{
+	const struct lp5562_config *config = dev->config;
+	const struct gpio_dt_spec *en_pin = &config->en_pin;
+	int err = 0;
+
+	/* if gpio control is enabled, we need to assert EN_GPIO first */
```

Comment:
Please use a capital letter to start sentence and a period to end it. We want to be consistent with other comments in the. Also please use GPIO instead of gpio (accronym).

My comment above applies to other parts.